app/scripts/models/reliers/oauth.js

```diff
@@ -330,7 +330,7 @@ function replaceItemInArray(array, itemToReplace, replaceWith) {
 }
 
 function scopeStrToArray(scopes) {
-  if (! _.isString) {
```

Comment:
Wow....

Reply:
Can you add a test for this?

Reply:
I'm also moving this function in the pairing PR into the base relier to make it testable and accessible. gotta figure out how to manage that...